src/main/kotlin/com/cognifide/gradle/aem/pkg/PrepareTask.kt

```diff
@@ -16,6 +16,9 @@ open class PrepareTask : AemDefaultTask() {
     @OutputDirectory
     val vaultDir = AemTask.temporaryDir(project, NAME, PackagePlugin.VLT_PATH)
 
+    @OutputDirectory
```

Comment:
it this task is always executed (`upToDateWhen { false } `) then you redundantly are using cache related annotations which have an impact to determine if task should be executed or not